sympy/utilities/lambdify.py

```diff
@@ -450,6 +455,48 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def should_dummify(expr, printer=None):
+    """
+    Returns the boolean value of dummify to be used.
+
+    Parameter
+    ---------
+    expr : Represents the expression for which dummify should be checked
+    printer : Represents the printer which is provided to this function
```

Comment:
'which is provided to this function' is inexpressive IMHO. Please improve this description here or describe the need for the `printer` kwarg in the summary in line 460.